bitnami/keycloak/values.yaml

```diff
@@ -1011,7 +1011,8 @@ metrics:
     ## @param metrics.service.ports.http Metrics service HTTP port
     ##
     ports:
-      http: 9000
+      metrics: 9000
```

Comment:
Thanks for your contribution. Please note you need to add the metadata for this new parameter, see the above metadata corresponding to the already existing parameters.

Reply:
I updated the metadata.